records/student.record.ts

```diff
@@ -60,7 +64,7 @@ export class StudentRecord implements StudentEntity {
 
     if(!obj.studentId){
 
-      if (checkGithubUsername(this.githubUsername) !== null) {
+      if ((checkGithubUsername(this.githubUsername) !== null)&&(this.githubUsername!=="")) {
```

Comment:
jeśli gituser nie zostanie podany to (checkGithubUsername(this.githubUsername) nie zwróci normalnie  null?

Reply:
Gdy admin dodaje studenta, to nie wpisuje gitName i DB wywala błąd. Dlatego ten dodatek

Reply:
ok